Postgres throws DiskFull during performance check

After inserting roughly 2 million `boefje_meta` and `raw_files` into Bytes and counting the number of raw files per organization, I get a
```
bytes_1                | [2023-04-18 09:09:57 +0000] [21] [ERROR] [sql_meta_repository] An error occurred: (psycopg2.errors.DiskFull) could not resize shared memory segment "/PostgreSQL.610088736" to 33554432 bytes: No space left on device
```

Also see https://stackoverflow.com/questions/56751565/pq-could-not-resize-shared-memory-segment-no-space-left-on-device as a reference. A greater `shm_size` seemed to resolve the issue. But as https://dba.stackexchange.com/questions/275378/dev-shm-size-recommendation-for-postgres-database-in-docker points out, another option would be to set `max_parallel_workers` to 0 for development to (hopefully) trade some performance for some fault tolerance and not eat away more memory resources. For the `docker-compose.release-example.yml` the latter is probably not a good idea though as this is not a good production setup, but also in general I think we should not be doing production resource tuning in this example file.

--- a/docker-compose.release-example.yml
+++ b/docker-compose.release-example.yml
@@ -16,6 +16,7 @@ services:
 
   postgres:
     image: postgres:15
+    shm_size: 256MB
     ports:
       - "0.0.0.0:5432:5432"
     volumes:

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -17,6 +17,7 @@ services:
   postgres:
     restart: unless-stopped
     image: postgres:15
+    shm_size: 256MB
     ports:
       - "127.0.0.1:5432:5432"
     healthcheck:

